deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml

```diff
@@ -40,7 +40,7 @@ spec:
           value: "ivo://cadc.nrc.ca/gms?skaha-users"
         - name: skaha.adminsgroup
           value: "ivo://cadc.nrc.ca/gms?skaha-admins"
-        image: images.canfar.net/skaha-system/skaha:0.9.21
+        image: images.canfar.net/skaha-system/skaha:0.9.22
```

Comment:
This PR causes an API change.  Let's make it version 0.10.0

Reply:
Good point!